<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUpdateInfoComponent</class>
 <widget class="QFrame" name="CUpdateInfoComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>108</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>swift update info</string>
  </property>
  <layout class="QGridLayout" name="gl_DistributionInfo" columnstretch="1,2,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="2">
    <widget class="QComboBox" name="cb_Platforms"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cb_Channels"/>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cb_ArtifactsPilotClient"/>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cb_ArtifactsXsb"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pb_DownloadXSwiftBus">
     <property name="toolTip">
      <string>XPlane only</string>
     </property>
     <property name="text">
      <string>download</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pb_DownloadInstaller">
     <property name="text">
      <string>download</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_CurrentVersion">
     <property name="text">
      <string>This version:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbl_ArtifactsSwift">
     <property name="text">
      <string>swift installer:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbl_ArtifactsXSwiftBus">
     <property name="toolTip">
      <string>Only required for XP</string>
     </property>
     <property name="text">
      <string>xswiftbus:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbl_CurrentVersionDisplay">
     <property name="text">
      <string>&lt;version goes here&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_Channel">
     <property name="text">
      <string>Channel/OS:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Info:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_StatusInfo">
     <property name="text">
      <string>&lt;status goes here&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QPushButton" name="pb_CheckForUpdates">
     <property name="text">
      <string>check again</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pb_CheckForUpdates</tabstop>
  <tabstop>cb_Channels</tabstop>
  <tabstop>cb_Platforms</tabstop>
  <tabstop>cb_ArtifactsPilotClient</tabstop>
  <tabstop>pb_DownloadInstaller</tabstop>
  <tabstop>cb_ArtifactsXsb</tabstop>
  <tabstop>pb_DownloadXSwiftBus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
